packages/components/config/rollup.config.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable camelcase */
+import nodeResolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import nodeGlobals from 'rollup-plugin-node-globals';
+import { terser } from 'rollup-plugin-terser';
+import postcss from 'rollup-plugin-postcss';
+import json from '@rollup/plugin-json';
+import { dependencies, peerDependencies, name } from '../package.json';
+import { createFilter } from '@rollup/pluginutils';
+import glob from 'glob';
+
+const globMapper = (mapper) => glob.sync(mapper).reduce((acc, item) => {
```

Comment:
I think that there can be some common place for this script.

Reply:
Mhh, yeah. I'll write some common config, since a lot of these things just repeat.